src/families/ethereum/types.js

```diff
@@ -56,6 +56,7 @@ export type Transaction = {|
   estimatedGasLimit: ?BigNumber,
   feeCustomUnit: ?Unit,
   networkInfo: ?NetworkInfo,
+  allowZeroAmount?: Boolean,
```

Comment:
it's a subtility but i think it should be `boolean`

Reply:
![Capture d’écran 2021-03-29 à 10 15 43](https://user-images.githubusercontent.com/211411/112807050-bd843c00-9077-11eb-9521-7c7e6f9283de.png)

https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVAXAngBwKZgCCYAvGAN6phgBGAXLXHDHgIYB2ANFWAEIO8mLDtwC+AbnQsMYVg2JlytBhgBOAVzyc+KjQQlA